readthedocs/proxito/tests/responses/v0.json

```diff
@@ -0,0 +1,120 @@
+{
+    "comment": "THIS RESPONSE IS IN ALPHA FOR TEST PURPOSES ONLY AND IT'S GOING TO CHANGE COMPLETELY -- DO NOT USE IT!",
+            "project": {
+                "created": "2019-04-29T10:00:00Z",
+                "default_branch": "master",
+                "default_version": "latest",
+                "id": 1,
+                "language": {
+                    "code": "en",
+                    "name": "English"
```

Comment:
`project.language.name` here is an example of where we could support a translated API response, probably based on accept headers. If the addition of a language aware API endpoint changes the implementation here drastically, maybe this is a now conversation. But I'm assuming it's not and we can come back to this as we give users more access into this response structure.